adr/006-domain-plugin.md

```diff
@@ -20,6 +20,26 @@ larger trusted intermediary application.  This also allows the domain to develop
 trusted intermediary operates.  E.g. Is the trusted intermediary in a Jetty Java container hosted on a VM or is it
 hosted in Azure Functions?
 
-### Related Issues
+## Impact
+
+### Positive
+
+Encapsulation of domain-specific logic: Ensuring isolation of concerns; Core application logic remains unaffected
+Reusability: Plugins can be reused reducing duplication
+Flexibility: Domains can evolve independently and potentially be extracted into separate projects.
+Targeted testing: Focused unit and integration testing
```

Comment:
These should either be bullet points or get an extra line break between them. Right now they're all running together